docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+# Long-term Roadmap
+
+This is the long-term, draft Roadmap. Note that this a vision document that reflects our
+current desires. It is not a commitment to implement everything listed in this roadmap.
+The primary purpose of this document is to ensure all contributors work in alignment.
+As our vision changes over time maintainers reserve the right to modify this roadmap,
+add and _remove_ items from it.
+
+Description|Status|Links|
+-----------|------|-----|
+**Testing**|
+Add metric testing to the testbed|Done|
+Implement matrix testing in the testbed and convert benchmarks for receivers and exporters to use it|Done|
+Implement E2E tests for all core formats (Jaeger, Zipkin, OpenCensus, OTLP, Prometheus)| |[#482](https://github.com/open-telemetry/opentelemetry-collector/issues/482)
+Improve comparison with baseline performance and fail CI build on degradations| |
+Add code coverage collection to Contrib repository| |
+Add 2-stage build pipeline: separate build and certification jobs| |
+| |
+**Performance**|
+Convert internal memory representation to OTLP Protobuf| |[#478](https://github.com/open-telemetry/opentelemetry-collector/issues/478)
+Graceful shutdown (pipeline draining)| |[#483](https://github.com/open-telemetry/opentelemetry-collector/issues/483)
```

Comment:
`Graceful shutdown` shouldn't be on performance, perhaps a section for robustness (it is "graceful" but it is a start to reduce and better record data losses in case of unexpected errors)

Reply:
Moving to separate "Other" section for now.

Reply:
Done.